### ISSUE TITLE
feat: ion-picker mouse scroll control

### ISSUE THREAD

**Issue by nelunull22123** [driftyco/ionic#29463]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already include this feature request, without success.

### Describe the Feature Request

I want a setting to prevent scrolling of many lines
Nowadays, selecting with the mouse wheel is almost impossible

### Describe the Use Case

The problem is that when using the mouse wheel, it is very difficult to select elements. They scroll across several lines at once and very quickly. It is necessary to prohibit or somehow limit the scrolling of elements (it is important to make this optional).

![chrome_4VGycGpKPT](https://github.com/ionic-team/ionic-framework/assets/109805785/ba682017-5ddd-4faa-b9d2-71bc6f8e0d93)

### Describe Preferred Solution

I think we need to add an option, when enabled, when scrolling, only the closest element will be selected and there will be a delay (for example, 300ms) and only after that the next element will be selected

**Comment by sean-perkins** [comment]
@nelunull22123 what is your scroll speed set to?

With the fastest speed enabled in MacOS settings I am still able to intentionally scroll to any option within the list with high precision: 

![CleanShot 2024-05-08 at 13 39 41@2x](https://github.com/ionic-team/ionic-framework/assets/13732623/e77f229e-cfa9-42cb-832e-fec96c7a2406)

It is difficult to tell from the recording provided in the issue if the scrolling is exaggerated when you are scrolling from the top and bottom repeatedly. I am using a logitech mouse with a mouse wheel:


https://github.com/ionic-team/ionic-framework/assets/13732623/5b6d74f0-c958-4c0e-982a-65a71481f589

CSS scroll snapping (the underlying implementation) does not allow for adjusting the scrolling speed and easing of the interaction.

**Comment by thetaPC** [comment]
Closing due to the lack of a reproduction. If the issue persists, please open a new one with a minimal reproduction. Thanks!

**Comment by NeluQi** [comment]
I changed my PC, now I'm running Windows 11.
I have a different mouse.

But the problem remains. I can't use the mouse wheel to select items.

**Comment by thetaPC** [comment]
@NeluQi Please create a new issue with a minimal reproduction so we determine the next steps. Thank you!
